src/thunderbots/software/util/parameter/parameter.h

```diff
@@ -26,12 +26,12 @@
  *
  * See http://wiki.ros.org/Parameter%20Server for the list of types
  *
- * In our codebase, we support bool, int32_t, double, string, and lists (aka vectors)
- * of the previous 4 types
- */
+ * In our codebase, we support bool, int32_t, double, and strings
+ * */
 
 namespace{
-	std::string kParameterNs = "/parameters";
+        // what namespace the parameters will be in (ROS param server)
+	std::string kNamespaceForParameters = "/parameters";
```

Comment:
Still a bit confused why there's a `k` at the start of this variable name? Does it mean something special (is it an abbreviation)?

Reply:
its just something we do at work to differentiate constants from other stuff, did it out of habit, removed it now (also added a const infront)

Reply:
lol got rid of it, never thought about it